### PR TITLE
Add new content for academic writing

### PR DESCRIPTION
**Pull request by universvm**
## Edited
- Links to articles for academic writing as per #511 
- Overview

## Added
- People note

## Checklist
- [X] before creating a new note, I searched the vault
- [X] new notes have the `.md` extension
- [X] (if applicable) attached images have descriptive file names
- [X] (if applicable) for new notes in the folder "04 - Guides, Workflows, & Courses", I added a link to them in one of the "for {group X}" overviews: https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

### CODE REVIEW

**Comment by universvm** [comment]
Hey @lguenth / @claremacrae , can I ask for feedback on this?

File: 01 - Community/People/🗂️ People.md [hunk 1]
```diff
@@ -517,6 +517,7 @@ Overviews of all the work from notable People like Plugins Developers, Theme Des
 -  [[01 - Community/People/twentytwokhz|twentytwokhz]]
 -  [[01 - Community/People/twibiral|twibiral]]
 -  [[01 - Community/People/tylernguyen|tylernguyen]]
+-  [[01 - Community/People/universvm|Leonardo Castorina]]
```
Comment (claremacrae) [inline 01 - Community/People/🗂️ People.md:520]: For future reference, these lists of files are machined-generated by script, and don't need to be updated manually.

You don't need to remove it now, though. We can deal with when we merge.

File: 01 - Community/People/universvm.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+%% Begin Hub: Released contributions %%
+### Guides
+- [Obsidian Tutorial for Academic Writing](https://betterhumans.pub/obsidian-tutorial-for-academic-writing-87b038060522) by [[universvm|Leonardo Castorina]]
+- [How to Boost your Productivity for Scientific Research](https://betterhumans.pub/how-to-boost-your-productivity-for-scientific-research-using-obsidian-fe85c98c63c8) by [[universvm|Leonardo Castorina]]
+
+## Sponsor this author
+- [Ko-Fi](https://ko-fi.com/leocastorina) ^ko-fi
+
+
+%% End Hub: Released contributions %%
```
Comment (claremacrae) [inline 01 - Community/People/universvm.md:30]: The idea behind the commented-out headings is that people un-comment the headings they have content for, and then insert their own links/content in those sections.

That way, all pages have broadly similar layout, and it is easier for us to update the pages by script in future.

It's not obvious, but the comments on lines 21 and 30 mark text which will be over-written by automated scripts.
The comment in line 32 is meant to indicate the start of manually added lists, such as the guides you added.
Please could you move the guides section down to just above the `### Others` section, which is current lines 38-40.

Also sorry if it isn't obvious, but any sponsorship links should go in the 'Sponsor this author' section which is currently commented-out on line 43, so:

1. Uncomment the 'Sponsor this author' section around the current line 43
2. Then insert your LoFi link alongside the other comment-out sponsorship links.

File: 01 - Community/People/universvm.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Leonardo Castorina
+tags:
+- 
+publish: true
+---
+
+# Leonardo Castorina
+
+- Publish sites: <https://medium.com/@universvm/> ^website
+- Twitter: [Twitter](https://twitter.com/leocastorina/) ^twitter-->
```
Comment (claremacrae) [inline 01 - Community/People/universvm.md:12]: When previewed in Obsidian, this line looks odd.

image


Please remove the `-->` split comment at the end of the line. Thank you.

**Review by claremacrae — CHANGES_REQUESTED** [review]
Many thanks for the contribution. It's not in my field, but I'm sure others will find it useful.

I checked the links and they are all good.
I previewed the pages in Obsidian, and compared the content against the comments in the author template.

I've requested a few changes as a result.

**Comment by claremacrae** [comment]
> Hey @lguenth / @claremacrae , can I ask for feedback on this?

Hi @universvm Thank you, I have reviewed the changes and made some suggestions.

I see you made your pull request directly off your main branch, rather than creating a branch. If you have any problems making the edits requested, please let us know and we will try and help you work through it.

**Comment by universvm** [comment]
Hi @claremacrae , thank you so much for your feedback, I made the changes you requested. 

Could you have a look when you have time, just to double check I've done it right? Thanks in advance!

**Comment by claremacrae** [comment]
Hi @universvm thank you for making the changes.

This has got a bit complicated because GitHub is complicated and the new changes are not in this pull request.

I could talk you through fixing it in 5 or 10 minutes on an audio call, but cannot see how to type up instructions that there would be any chance of following here.

I don't suppose you would be willing to have an an audio call please at some mutually convenient time?
My email address is in my GitHub bio.

**Comment by universvm** [comment]
Hi @claremacrae , that's weird, I can perhaps close the PR and open a new one with the correct branch?

I can email you in a bit, happy to chat whenever you have time :)

**Comment by universvm** [comment]
closed in favor #530

**Pull request closed without merging**